Core/src/org/sleuthkit/autopsy/logicalimager/dsp/LogicalImagerPanel.java

```diff
@@ -482,6 +482,9 @@ private void refreshButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN
         int i = 0;
         for (File root : roots) {
             String description = FileSystemView.getFileSystemView().getSystemTypeDescription(root);
+            if ("Network Drive".equalsIgnoreCase(description)) { // NON-NLS
```

Comment:
Will this work on a non-English version of windows?

Reply:
I don't know. I can't find a solution that doesn't test for "Network Drive".

Reply:
@APriestman I found this https://gist.github.com/digulla/31eed31c7ead29ffc7a30aaf87131def which spawn a process with "CMD /c net use Z:" to test for network drive.

Is this solution OK? Requires CMD and net available on the target system (which should be available on all Windows system)